README.md

```diff
@@ -1,6 +1,10 @@
-# java-gradle-starter-project
+# GPS Logger project
 
-Here is a starter project for the ArcGIS Maps SDK for Java with Gradle. 
+This project is a demonstration of an IoT GPS data logger which integrates into the ArcGIS platform.
+
+It has been designed to work on a Raspberry Pi Model 3B+ (or higher model) with a USB GPS receiver.
+
+The project uses an early adopter version of the ArcGIS Maps SDK for Java specifically compiled to work with Arm Linux platforms such as the Raspberry Pi or Jenson Orin units.  The Maps SDK for Arm Linux can be requested by emailing ARM64LinuxNative@esri.com.  This will install the libraries needed for this app in a Maven Local instance.  This is a temporary way of getting the SDK whilst it is in Beta.  Subsequent releases will be available in a public maven repository.
```

Comment:
Lines 11 to 13 I think are inaccurate? Could probably remove the screenshot file too?

```
The app launches a window displaying a map.
![screenshot](screenshot.png)
```

Reply:
Agreed, these have been removed